libraries/TDataParser/TDataParser.cxx

```diff
@@ -57,13 +57,13 @@ int TDataParser::TigressDataToFragment(uint32_t* data,int size,unsigned int mida
 	if(!SetTIGTriggerID(dword,EventFrag)) {
 		delete EventFrag;
 		printf(RED "Setting TriggerId (0x%08x) failed on midas event: " DYELLOW "%i" RESET_COLOR "\n",dword,midasSerialNumber);
-		return NumFragsFound;  
+		return -x;  
```

Comment:
Is this change intentional? Or did it slip in? This is more like what GRIFFIN has - just wondering why it wasn't this to begin with.